src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -5,15 +5,15 @@
 import com.cloudbees.jenkins.plugins.sshcredentials.SSHUserPrivateKey;
 import com.cloudbees.plugins.credentials.common.StandardCredentials;
 import com.cloudbees.plugins.credentials.common.StandardUsernamePasswordCredentials;
-
+import com.google.common.collect.Lists;
```

Comment:
Please don't reorder the import statements.  It is a pure whitespace change that increases the risk that other pull requests will then conflict with this pull request.

In a perfect world, there would be very few pull requests pending and we could change white space freely.  Unfortunately, I'm far behind reviewing pull requests and need to limit white space changes.

Reply:
Well the import ordering is all over the place 🤷‍♂️ so I have to do some handheld import handling 🤭

Reply:
"All over the place" is a very kind way of saying "a total mess".  Yes, I agree.  Unfortunately, we need to minimize the impact of past mistakes on pending pull requests, so we keep whitespace changes to a minimum.  Sorry for the extra work (outside the IDE, etc.).

Reply:
Well then at least outside of any ordered list 😅